src/middlewares/verifyUser.js

```diff
@@ -0,0 +1,19 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+import customize from '../helpers/Customize';
+
+dotenv.config();
+
+const verifyUser = (req, res, next) => {
```

Comment:
@hezronkimutai I think you should name this function to `verifyToken`. it is good to be specific.

Reply:
Okey Eric, actually its like we don't need that  file because we already have Authenticate token.. let me delete it, thanks.